docs/getting-started/installation/frc-plugin.md

```diff
@@ -0,0 +1,6 @@
+# Installing the FRC plugin for IntelliJ
+To get things like the right build tasks, class templates, and project templates in IntelliJ you need the FRC plugin. To install this you need to open IntelliJ, and navigate to `File > Settings > Plugins` and then click on the "Marketplace" tab. You're going to want to search "FRC" and download the plugin by Mark Vedder (also shows as Javaru for some people).
+
+Once you've installed it you'll now be able to create a robot project yourself buy going to `File > New > Project` and selecting "FRC Robot Project". You'll be prompted to select things like your chosen template, language, and WPILib version, but all of that is pretty self explanatory if you're familiar with the WPILib VS Code plugin.
+
+And that's it! If you've gotten to this point you are now free to start developing FRC robot projects using kotlin!
```

Comment:
> If you've gotten to this point

I personally think we could do without the 'fluff', just to give the docs a more 'official' tone. Feel free to disregard if you like this